src/main/java/mekanism/common/integration/computer/TableType.java

```diff
@@ -0,0 +1,73 @@
+package mekanism.common.integration.computer;
+
+import com.mojang.serialization.Codec;
+import com.mojang.serialization.codecs.RecordCodecBuilder;
+import java.util.Collections;
+import java.util.LinkedHashMap;
+import java.util.Map;
+import mekanism.common.util.MekCodecs;
+
+/**
+ * Defines the format of a "table" (Map) format of a Mekanism data structure
+ */
+public record TableType(String description, String humanName, Map<String, FieldType> fields, Class<?> extendz) {
```

Comment:
Instead of using `extendz` to make it so that it doesn't clash with the keyword `extends`, how do you feel about using `extended`?